keyboards/a_jazz/akc084/readme.md

```diff
@@ -0,0 +1,25 @@
+# A-JAZZ  AKC084
+
+![](https://github.com/fenggx-a-jazz/documents/blob/main/akc084.jpg?raw=true)
+![](https://github.com/fenggx-a-jazz/documents/blob/main/akc084-2.png?raw=true)
+![](https://github.com/fenggx-a-jazz/documents/blob/main/akc084-2.jpg?raw=true)
```

Comment:
Please rehost these images on Imgur.

Reply:
I will remove the images for now. Setup imgur soon.